translations/harbour-barcode-pt_PT.ts

```diff
@@ -0,0 +1,378 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="pt_PT">
```

Comment:
I would just say "pt" as long there's no separate translations for Portugal and other Portuguese-speaking countries.

Reply:
I can fix that myself though.